.github/dependabot.yml

```diff
@@ -48,8 +38,7 @@ updates:
     - "/actions/update-issue/"
     - "/actions/upload-repository/"
```

Comment:
Unrelated, but may I ask why all `actions/` subdirs are listed explicitly? Given the findings (Dependabot recursing into subdirs) that led to the current PR, would `/` or `/actions` be enough?